### DISCUSSION TITLE
Why can't I see  Github Pages as normal?

### DISCUSSION THREAD

**Discussion by ZhuZheart98** [community/community#22556] [Other Feature Feedback, Questions, & Ideas]
<p>when I open pages, the pages was normal. <strong>But when I see a project, it can’t show nomally</strong>. And it need more time to load page than normal. </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/6a0afa34fce0c2a3034b1619a44e53d1f5123c1b.png" data-download-href="/uploads/short-url/f8686HIqNJ61FNaEkGvrNFBBfFF.png?dl=1" title="1.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/35882447/181071965-3b82ce25-86ae-4b37-ae82-6992b4c6f258.png" alt="1.PNG" data-base62-sha1="f8686HIqNJ61FNaEkGvrNFBBfFF" width="690" height="299" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/6a0afa34fce0c2a3034b1619a44e53d1f5123c1b_2_690x299.png, https://aws1.discourse-cdn.com/github/optimized/1X/6a0afa34fce0c2a3034b1619a44e53d1f5123c1b_2_1035x448.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/6a0afa34fce0c2a3034b1619a44e53d1f5123c1b_2_1380x598.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/6a0afa34fce0c2a3034b1619a44e53d1f5123c1b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">1.PNG</span><span class="informations">1915×832 63.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div> <div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/a81455b0f644c47518b56d1c57985f0189756499.png" data-download-href="/uploads/short-url/nYTMtZHYeGR3o99rlh4LuwNSj7X.png?dl=1" title="2.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/35882447/181071966-46a886b9-612e-4eb3-aca5-56b2701bf843.png" alt="2.PNG" data-base62-sha1="nYTMtZHYeGR3o99rlh4LuwNSj7X" width="690" height="313" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/a81455b0f644c47518b56d1c57985f0189756499_2_690x313.png, https://aws1.discourse-cdn.com/github/optimized/1X/a81455b0f644c47518b56d1c57985f0189756499_2_1035x469.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/a81455b0f644c47518b56d1c57985f0189756499_2_1380x626.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/a81455b0f644c47518b56d1c57985f0189756499_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">2.PNG</span><span class="informations">1879×853 55.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div> <div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/fb87e0c4f54cd16bb6664b1050a39aed34f41d47.png" data-download-href="/uploads/short-url/zT8XFkTo5HnE6LPsIw2v3NahXcX.png?dl=1" title="3.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/35882447/181071967-37a999f3-d10c-41c3-a8af-e31b3bf5459c.png" alt="3.PNG" data-base62-sha1="zT8XFkTo5HnE6LPsIw2v3NahXcX" width="690" height="320" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/fb87e0c4f54cd16bb6664b1050a39aed34f41d47_2_690x320.png, https://aws1.discourse-cdn.com/github/optimized/1X/fb87e0c4f54cd16bb6664b1050a39aed34f41d47_2_1035x480.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/fb87e0c4f54cd16bb6664b1050a39aed34f41d47_2_1380x640.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/fb87e0c4f54cd16bb6664b1050a39aed34f41d47_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">3.PNG</span><span class="informations">1891×879 69.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by ZhuZheart98** [comment, marked as answer]
<p>I find the reason is the .css or .js file can’t be load. I have tried several ways, so lucky, I find a way.</p>
<p>1.“win+R”, input ‘cmd’，so we get in this <img src="https://user-images.githubusercontent.com/35882447/181071969-3dd45251-b037-401f-9b85-2acf63105944.png" alt="捕获1.PNG" data-base62-sha1="dEDkEw7P8XvwTTAr7SKlCj0gsqU" width="553" height="98"></p>
<ol start="2">
<li>we input’ping <a href="http://github.global.ssl.fastly.net" rel="nofollow noopener">github.global.ssl.fastly.net</a>’</li>
</ol>
<p>then it will show like this. </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/1/16929f4402fe1881826f1f6c258e816d28aea620.png" data-download-href="/uploads/short-url/3dGDJiQu2y7IajMnwchFucakRMI.png?dl=1" title="捕获1.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/35882447/181071971-945516ec-3d1a-4717-bd9d-5ab8de994749.png" alt="捕获1.PNG" data-base62-sha1="3dGDJiQu2y7IajMnwchFucakRMI" width="690" height="226" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/1/16929f4402fe1881826f1f6c258e816d28aea620_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">捕获1.PNG</span><span class="informations">765×251 11.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>3.modify your hosts file, add this </p>
<p>151.101.1.194 <a href="http://github.global.ssl.fastly.net" rel="nofollow noopener">github.global.ssl.fastly.net</a></p>
<p>4.when I get in <a href="http://github.com" rel="nofollow noopener">github.com</a>, it works.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/8/82cf7b67a7fac8b19073bdb5942fd18a51c0111e.png" data-download-href="/uploads/short-url/iFcBa66kHO14JIKzVLl9Yhu3AGW.png?dl=1" title="捕获1.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/35882447/181071972-7ff44916-ff19-4613-a7cb-504b5ce731f3.png" alt="捕获1.PNG" data-base62-sha1="iFcBa66kHO14JIKzVLl9Yhu3AGW" width="690" height="323" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/8/82cf7b67a7fac8b19073bdb5942fd18a51c0111e_2_690x323.png, https://aws1.discourse-cdn.com/github/optimized/2X/8/82cf7b67a7fac8b19073bdb5942fd18a51c0111e_2_1035x484.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/8/82cf7b67a7fac8b19073bdb5942fd18a51c0111e_2_1380x646.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/8/82cf7b67a7fac8b19073bdb5942fd18a51c0111e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">捕获1.PNG</span><span class="informations">1884×883 109 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by AndreaGriffiths11** [comment]
<p><a class="mention" href="/zhuzheart98">@zhuzheart98</a> Thank you for coming back and sharing your solution! We’re looking forward to seeing you around!</p>

**Comment by ZhuZheart98** [comment]
<p>You’re welocome. I will.</p>
